docker/api/network.py

```diff
@@ -19,16 +19,30 @@ def networks(self, names=None, ids=None):
         return self._result(res, json=True)
 
     @minimum_version('1.21')
-    def create_network(self, name, driver=None):
+    def create_network(self, name, driver=None, ipam=None):
         data = {
             'name': name,
             'driver': driver,
+            'ipam': ipam,
         }
         url = self._url("/networks/create")
         res = self._post_json(url, data=data)
         return self._result(res, json=True)
 
     @minimum_version('1.21')
+    def generate_ipam_config(self, driver='default', subnet=None,
```

Comment:
Also, it would be nice to have it renamed to `create_ipam_config` to be consistent with the currently existing `create_host_config` and `create_container_config`. :smile: